crates/turborepo-lib/src/cli.rs

```diff
@@ -588,6 +588,14 @@ pub async fn run(
         command
     } else {
         let run_args = mem::take(&mut cli_args.run_args).ok_or(anyhow!("No command specified"))?;
+        if run_args.tasks.is_empty() {
+            let mut cmd = <Args as CommandFactory>::command();
+            match cmd.print_help() {
+                Ok(_) => process::exit(1),
+                Err(_) => process::exit(1),
+            }
```

Comment:
Since you are just ignoring this and dying regardless, you can write this as
```suggestion
            let _ = cmd.print_help();
            process::exit(1);
```

Alternatively, if you really like the current match style, use an "or" pattern:
```suggestion
            match cmd.print_help() {
                Ok(_) | Err(_) => process::exit(1),
            }
```